### ISSUE TITLE
Prioritize long press characters on French keyboard 

### ISSUE THREAD

**Issue by ibrahimasarr** [AnySoftKeyboard/LanguagePack#138]
Long press on many characters displays alternative characters in a weird order. For instance 
- "a" displays 1àá@âã. 
- "e" displays 3èé€ê
There should be some logic in order of appearance. The a should display in order of character frequency: aà@...
The e eéèê€...
It is annoying to have numbers first.
Thanks

**Comment by ibrahimasarr** [comment]
I am only taking about French keyboard where the accented characters are very frequent. Thanks.

**Comment by lotth** [comment]
Yes, that is the one I considered too. I used another keyboard for a while and I found this default behaviour very practical. That is actually one feature that made me choose AnySoftKeyboard over AOSP.
Maybe it is just something to get used to (either way).
It may also help if one can choose the long press delay in order to access accented characters faster, I don't know it this is possible with AnySoftKeyboard.
